src/vs/workbench/contrib/quickaccess/browser/commandsQuickAccess.ts

```diff
@@ -227,4 +231,15 @@ export class ClearCommandHistoryAction extends Action2 {
 	}
 }
 
+//#region --- Register a command to get all actions from the command palette
+CommandsRegistry.registerCommand('_getAllCommands', async function (accessor: ServicesAccessor) {
```

Comment:
Why do we need this? It is somewhat ugly that we instantiate a UI component here only to get to the commands. Maybe the functionality for getting the commands should better be extracted to its own function or class?